poollet/machinepoollet/client/client.go

```diff
@@ -97,3 +97,20 @@ func SetupLoadBalancerRoutingNetworkRefNameField(ctx context.Context, indexer cl
 		},
 	)
 }
+
+const MachineMachinePoolRefNameField = "machine-machinepoolref-name"
```

Comment:
We don't need an indexer for this - the `machinePoolRef.Name` field of a machine is indexed in the `onmetal-apiserver`. You can simply use `computev1alpha1.MachineMachinePoolRefNameField`.